worlds/metroidprime/__init__.py

```diff
@@ -113,9 +115,22 @@ def set_rules(self) -> None:
         self.multiworld.completion_condition[self.player] = lambda state: (
             state.can_reach("Mission Complete", "Region", self.player))
 
-    def generate_output(self) -> None:
-        configjson = make_config(self, self.options)
-        py_randomprime.patch_iso(self.settings.rom_file, "prime_out.iso", configjson)
+    def generate_output(self, output_directory: str) -> None:
+        configjson = make_config(self)
+        # convert configjson to json
+        import json
+
+        configjsons = json.dumps(configjson, indent=4)
```

Comment:
I left this in for now just to make debugging the config easier, I'm good to remove if you want or to flag it so it only does it if we have a debug argument passed in or something